mopidy/audio/actor.py

```diff
@@ -572,9 +573,13 @@ def _on_source_setup(self, element, source):
         else:
             self._appsrc.reset()
 
+        if self._live_stream and source.__class__.__name__ == "GstSoupHTTPSrc":
```

Comment:
Don't check for the element type, do like https://github.com/mopidy/mopidy/blob/develop/mopidy/audio/utils.py#L65 and just check for the live property.

Reply:
Done